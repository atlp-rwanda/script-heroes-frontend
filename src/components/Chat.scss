
  h2 {
      text-align: center;
  }
  .container-login {
    margin-top: 10px;
  }
.chat {
    background-color: #e9ecef;
    height: 400px;
    padding: 10px;
    box-shadow: 0px 0px 10px gray;
  }
  .chat_image {
    width: 100%;
  }
  .container-chat {
    margin-top: 10px;
  }
  .chat_field {
    float: left;
    width: 87%;
    margin-top: -3px;
    font-size: 12px;
  }
  .send_image {
    width: 35px;
    height: 35px;
    float: right;
    margin-right: 35px;
    border-radius: 10px;
  }
  .send_image:hover {
    cursor: pointer;
  }
  .send {
    width: 103%;
    position: absolute;
    bottom: 10px;
}
.messages {
  height: 87%;
  word-wrap: break-word;
  overflow-y: scroll;
}
.bigText {
  font-weight: bold;
  font-size: 15px;
}
.mediumText {
  font-size: 13px;
}
.smallText {
  font-size: 10px;
  color: gray;
}
.chat_msg {
  padding: 7px;
  border-bottom: gray 1px solid;
  border-left: transparent 5px solid;
}
.chat_msg:hover {
  border-left: gray 5px solid;
  background-color: rgb(245, 235, 217);
}
