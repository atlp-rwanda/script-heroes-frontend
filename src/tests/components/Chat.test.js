import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import Chat from "../../components/Chats";

describe("Chat page", () => {
  it("Should match snapshot", () => {
    const wrapper = shallow(<Chat />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
