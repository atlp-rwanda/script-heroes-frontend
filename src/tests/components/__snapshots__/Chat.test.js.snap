// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat page Should match snapshot 1`] = `
<div>
  <BrowserRouter>
    <Container
      className="container-chat"
      tag="div"
    >
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          md="6"
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <h2>
            Chat
          </h2>
          <img
            alt="Chat Image"
            className="chat_image"
            src="test-file-stub"
          />
        </Col>
        <Col
          className="chat"
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <div
            className="messages"
          />
          <div
            className="send"
          >
            <Input
              autoComplete="off"
              className="chat_field"
              name="message"
              onKeyDown={[Function]}
              placeholder="Tape a message..."
              required={true}
              type="text"
            />
            <img
              alt="Send Image"
              className="send_image"
              onClick={[Function]}
              src="test-file-stub"
            />
          </div>
        </Col>
      </Row>
    </Container>
  </BrowserRouter>
</div>
`;
